.github/workflows/label-notify.yml

```
@@ -23,5 +23,5 @@ jobs:
                   team/security=@dcomas
                   team/dev-experience=@taylorsperry @jhchabran @kstretch9
                   team/repo-management=@jplahn @dan-mckean
-                  team/devops=@sourcegraph/cloud-devops
+                  team/devops=@daxmc99 @JenRed777 @danieldides
```

Should revert this change
```suggestion
                  team/devops=@sourcegraph/cloud-devops
```